recorder/src/main/cpp/circular_queue.hh

```diff
@@ -41,7 +41,7 @@ struct TraceHolder {
 class CircularQueue {
 public:
     explicit CircularQueue(QueueListener &listener, std::uint32_t maxFrameSize)
```

Comment:
Its used a few lines below. The JVMPI AGCT object size is allocated looking at this.